modules/install-central-operator.adoc

```diff
@@ -78,13 +78,29 @@ spec:
 |Use this parameter to configure Central to run only on specific nodes. See the `central.tolerations` parameter in the "Public configuration file" section in "Installing Central services for {product-title-short} on {osp}".
 |===
 * *Scanner Component Settings*: Settings for the default scanner, also called the StackRox Scanner. See the "Scanner" table in the "Public configuration file" section in "Installing Central services for {product-title-short} on {osp}".
-* Scanner V4 Component Settings: Settings for the optional Scanner V4 scanner, available in version 4.4 and later. It is not currently enabled by default. You can enable both the StackRox Scanner and Scanner V4 for concurrent use. See the "Scanner V4" table in the "Public configuration file" section in "Installing Central services for {product-title-short} on {osp}".
+* *Scanner V4 Component Settings*: Settings for the optional Scanner V4 scanner (Technology Preview), available in version 4.4 and later. It is not currently enabled by default. You can enable both the StackRox Scanner and Scanner V4 for concurrent use. See the "Scanner V4" table in the "Public configuration file" section in "Installing Central services for {product-title-short} on {osp}".
++
+--
+:FeatureName: Scanner V4
+include::snippets/technology-preview.adoc[]
+--
 +
 When Scanner V4 is enabled, you can configure the following options:
++
+[cols="2",options="header"]
```

Comment:
Suggest adjusting the columns widths to give more room to the description column